ruff.toml

```
@@ -1,6 +1,7 @@
 [lint]
 extend-select = [
 	"C901",
+	"RUF100",
 	"W",
```

One way I'm thinking of dealing with this situation is to create a separate section for local rules:

```suggestion
	"C901",
	"W",

	# local
	"RUF100",
```

---

Let's try that and see where it gets us.